packages/app/src/cli/models/app/loader.ts

```diff
@@ -931,13 +932,15 @@
     schemaForConfigurationFile = appSchema
   }
 
+  console.log('before')
```

Comment:
These console.log statements need to be removed, I assume they were committed accidentally 😉